packages/components/Button/src/useButton.ts

```diff
@@ -11,7 +11,7 @@ export const useButton = (props: ButtonProps): ButtonState => {
   // GH #1336: Set focusRef to null if button is disabled to prevent getting keyboard focus.
   const focusRef = disabled ? null : componentRef;
```

Comment:
driveby comment, why are we only filling the ref when the button is enabled?  The focus() function can no-op and avoid state changes that may invoke re-renders of parent components.  Should we open an issue to change this?

Reply:
The issue that the comment links to notes that this is a workaround: https://github.com/microsoft/fluentui-react-native/issues/1336 and the initial PR, https://github.com/microsoft/fluentui-react-native/pull/1332

Reply:
Basically, something on the MacOS' onPress hookup seems to not be working properly so they do this instead in the meantime.

Reply:
reading comprehension failure with a high contrast theme disabling syntax highlighting, unfortunate